### ISSUE TITLE
Add support for DM97MC and DZ6VS

### ISSUE THREAD

**Issue by coolguyconnor** [zlangbert/ha-daikinone#18]
Hi,

Just got new equipment installed in my home a few days ago which includes the DM97MC modulating gas furnace and DZ6VS heat pump. All controlled by the One Touch thermostat. So far the integration is working great but I would still like to contribute what I can to improving the integration and ensure my equipment is fully supported.

Diagnostics file from the thermostat page is attached. For privacy purposes I have removed the dealer contact info but can provide it if needed.

[daikinone-83c2c3f96df80af9b970031260c5cab7-Hallway Thermostat-3f8a9e453a0652823d63f09bc7067414.json.txt](https://github.com/zlangbert/ha-daikinone/files/14152665/daikinone-83c2c3f96df80af9b970031260c5cab7-Hallway.Thermostat-3f8a9e453a0652823d63f09bc7067414.json.txt)

**Comment by zlangbert** [comment]
Hi @coolguyconnor, glad to hear its working well for you. Your equipment should be well supported, but happy to double check things and see if there are any diagnostics we can add.

In addition to your data dump (thank you), if you can grab some pictures of the thermostats equipment status I can see if there is any data being reported that we aren't mapping.

Pictures of the following screens will tell us more: 
```
Thermostat Settings -> Dealer Edit -> Being Setup -> System Optimization -> Status -> Heat Pump/Air Handler/Furnace
```

(You can get your dealer code from the `i` in the top right of the dealer edit screen.)

**Comment by coolguyconnor** [comment]
[Photos-001.zip](https://github.com/zlangbert/ha-daikinone/files/14215764/Photos-001.zip)

Photos are attached. I had the furnace in emergency heat mode temporarily as the temperature was much lower than usual due to some work I was doing.

**Comment by zlangbert** [comment]
@coolguyconnor just released v0.6.0 which adds support for EEV coils. You should have a new device pop up with that data. Give it a try and let me know if it looks okay. I don't have a Fit system so can't test.

**Comment by zlangbert** [comment]
Thanks for testing, going to close this now. If you run into any issues let me know!
